receiver/k8sclusterreceiver/config.go

```diff
@@ -18,11 +18,14 @@ import (
 	"time"
 
 	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/kubernetes"
 )
 
 // Config defines configuration for kubernetes cluster receiver.
 type Config struct {
 	configmodels.ReceiverSettings `mapstructure:",squash"`
+	kubernetes.K8sAPIConfig       `mapstructure:",squash"`
```

Comment:
Looks good, thanks for updating observer.

Should we just call this `kubernetes.APIConfig` since it already has kubernetes in the package name?

Reply:
Yes, I think that's a good point. Will update.